account_payment_return/views/payment_return_view.xml

```diff
@@ -43,8 +44,8 @@
                                                     ('account_id.internal_type', '=', 'receivable'),
                                                     ('reconciled', '=', True)]"
                                             />
-                                    <field name="amount"/>
-                                    <field name="expense_amount" />
+                                    <field name="amount" sum="Total amount"/>
+                                    <field name="expense_amount" sum="Total expense amount"/>
```

Comment:
The string for `expense_amount` is `Charges Amount`, so it's better to use the same vocabulary for the total.

Reply:
Done, thanks!